src/client/components/home.js

```diff
@@ -1,9 +1,202 @@
+      ])
+    ]);
+  }
+}
 
 class Home extends Component {
+  constructor(props){
+    super(props);
+
+    this.bus = new EventEmitter();
+  }
+
   render(){
-    return h('div.home', 'TODO');
+    const CTA = () => {
+      return h(Popover, {
+        tippy: {
+          html: h(RequestForm, { bus: this.bus }),
+          onHidden: () => this.bus.emit('closecta')
+        }
+      }, [
+        h('button.home-cta-button.salient-button', 'Get started')
+      ]);
+    };
+
+    return h('div.home', [
+      h('div.home-banner', [
+        h('div.home-banner-overlay'),
+        h('div.home-corner-logo'),
+        h('div.home-banner-logo', [
+          h('h1.home-banner-title', 'Biofactoid'),
+          h('div.home-banner-tagline', 'Summarise.  Share.  Discover.')
+        ]),
+        h('div.home-cta.home-banner-cta', [
+          h(CTA)
+        ])
+      ]),
+      h('div.home-section.home-figure-section', [
+        h('div.home-figure.home-figure-summarize'),
+        h('div.home-caption', [
+          h('h2', 'Summarize'),
+          h('p', `Create a cloud-computable summary of your subcellular biology article with the easy-to-use Biofactoid app.  It takes only a few minutes, and it allows you to clearly and concisely summarize the heart of your article.`)
+        ])
+      ]),
+      h('div.home-section.home-figure-section', [
+        h('div.home-figure.home-figure-share'),
+        h('div.home-caption', [
+          h('h2', 'Share'),
+          h('p', `Share a compelling visual abstract that researchers can grasp at a glance.  Things like attributions and cross-references are all automatically handled for you.  Best of all, you and your colleagues can share your article's summary with just one click.`)
+        ])
+      ]),
+      h('div.home-section.home-figure-section', [
+        h('div.home-figure.home-figure-discover'),
+        h('div.home-caption', [
+          h('h2', 'Discover'),
+          h('p', `Find relevant articles and others in the community with shared interests.  Each article on Biofactoid is automatically semantically represented, indexed, and cross-referenced.  Biofactoid will allow researchers like you to explore subcellular research freely and seamlessly.`)
+        ])
+      ]),
+      h('div.home-section.home-text-section', [
+        h('div.home-cta.home-footer-cta', [
+          h(CTA)
+        ]),
+        h('div.home-credits', [
+          h('p.home-credit-logos', [
+            h('i.home-credit-logo'),
+          ]),
+          h('p.home-credit-text', [
+            'Biofactoid is freely brought to you in collaboration with ',
+            h('a.plain-link', { href: 'https://baderlab.org' }, 'Bader Lab at the University of Toronto'),
+            ', ',
+            h('a.plain-link', { href: 'https://sanderlab.org' }, 'Sander Lab at Harvard'),
+            ', and the ',
+            h('a.plain-link', { href: 'https://sanderlab.org' }, 'Pathway and Omics Lab at the University of Oregon'),
+            '.  ',
```

Comment:
I'm not completely sure where Emek wants us to point this.

Reply:
Let's just link to the university's homepage until he gives us an alternate link.